### DISCUSSION TITLE
Must Remote and Stream be running on same device?

### DISCUSSION THREAD

**Discussion by charlesglorioso** [GerryDazoo/Slinger#309] [General]
I am running the server on a Windows PC, and when using VLC and a Chrome browser both on that same PC, the stream and remote control both work fine.  When I run the stream in VLC on an IPAD, and the remote on the same IPAD, that also works, though running two programs at the same time on n IPAD is quite cumbersome.  However if I attempt to run the stream in VLC on the IPAD, and the remote on an iPhone, all running on the same Wi-Fi network, the server app shows that it is receiving commands from the remote, but those commands do not affect the stream.  (I attempted the two device setup since that is much more convenient than attempting two apps on the IPAD.)  Is there a code requirement that the remote and stream be running on the same device?

Separate issue. The stream in VLC sometimes comes up with sound, but no video.  Is this a format issue, or is there a more reliable player than VLC?

**Comment by blasco403** [comment]
I have issues with VLC sometimes as well. I use MPlayer on iOS, works well.

I regularly run the stream to a device (iPad/firetv) and then launch the remote on my iPhone with no issue.

**Comment by GerryDazoo** [comment]
There is no requirement for the media player and remote to be on the same device. I often use my phone as a remote control while streaming on my firestick.

**Reply by RandomMan719** [reply]
But if you have RemoteLock=Yes (config.ini) it does have to be on the device that started the stream, correct?
